routes/distribution.js

```diff
@@ -21,10 +21,9 @@ router.get("/api/distributions/:distribution_id", (req, res) => {
 //get all distributions in a plan
 router.get("/api/distributionsByPlan/:plan_id", (req, res) => {
   const plan_id = req.params.plan_id;
-  plans
-    .findById(plan_id)
-    .populate({ path: "distribution_ids" })
-    .then((plan) => returnData(plan.distribution_ids, res))
+  distributions
+    .find({ plan_id: req.params.plan_id })
+    .then((distributions) => returnData(distributions, res))
```

Comment:
should we also populate the fine req objects or is there another route that returns the fine reqs?

Reply:
fine req is a schema that we added so I think we should return fine reqs too, good catch!